CONTRIBUTING.md

```diff
@@ -150,5 +150,15 @@ By making a contribution to this project, I certify that:
 ### Testing Your Code
 Please visit the [testing doc](tests/README.md) for details about how to run tests
 
+
+### Pre-commit hooks
+We provide a basic config `.pre-commit-config.yaml` for [pre-commit](https://pre-commit.com/) to automate some aspects of the development process. As a convenience you can enable automatic copyright year updates by following the installation instructions on the pre-commit homepage, and setting the envrionment variable:
```

Comment:
Please wrap to within 100 characters for improved readability in a terminal and easier line-by-line commenting during reviews.

Also I'd like to see the phrase "pre-commit homepage" made into a link to the homepage.  (Yes, it's earlier in the paragraph, but this is closer to the "follow the installation instructions" directive.)